register.php

```diff
@@ -197,14 +198,14 @@
       </select>
   </div>
     <div class="form-group col-md-6">
-      <label for="inputCity">DOB</label>
+      <label for="inputCity"><?php echo $lang['REGISTER_DOB']; ?></label>
```

Comment:
@sajalbansal2298 please check, age validation isn't working.

Reply:
@dheeth It's done, Please review